packages/amplify_storage_plugin_interface/test/types/DownloadFile/downloadFileRequest_test.dart

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import 'package:amplify_storage_plugin_interface/src/DownloadFile/DownloadFileRequest.dart';
```

Comment:
We have been using PascalCase for directories and files. See https://dart.dev/guides/language/effective-dart/style

Reply:
That is the correct approach, but it seems like in Auth, Analytics and Storage are not using pascal case for files and folders in the `src/` directory but rather this pattern. 
I've started on a separate PR to do the renaming across plugins, since it pertains to the entire repo.